lmnet/tests/executor_tests/test_build_tfds.py

```diff
@@ -0,0 +1,194 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import numpy as np
+import pytest
+
+from executor.train import run as train_run
+from executor.build_tfds import run
+from lmnet import environment
+from lmnet.datasets.dataset_iterator import DatasetIterator
+from lmnet.datasets.tfds import TFDSClassification, TFDSObjectDetection
+from lmnet.utils import config as config_util
+
+
+_RUN_AS_A_SCRIPT = False
+
+
+# Apply reset_default_graph() in conftest.py to all tests in this file.
+# Set test environment
+pytestmark = pytest.mark.usefixtures("reset_default_graph", "set_test_environment")
+
+
+def setup_dataset(dataset_class, subset, **kwargs):
+    dataset = dataset_class(subset=subset, **kwargs)
+    return DatasetIterator(dataset, seed=0)
+
+
+def _show_images_with_boxes(images, labels):
```

Comment:
I'm not sure if we should show images during a test. we cannot automatically verify this output.

Reply:
This was for debugging this script. I've removed it.